lib/generators/rolemodel/saas/devise/templates/app/views/admin/users/index.html.slim

```diff
@@ -0,0 +1,27 @@
+header.header--index-action
+  h1 Users
+  - if Devise.mappings[:user].try(:invitable?)
+    = link_to 'Add User', new_invitation_path(:user), class: 'btn btn--primary'
+  = link_to 'Sign out', destroy_session_path(current_user), method: :delete
+
+.table__wrapper
+  table.table data-controller="clickable-rows"
```

Comment:
Clickable Rows is a Stimulus controller from BGEA right? I don't think it is something we've brought into the generator yet, or have we?

Reply:
good catch - yeah, that wasn't intended to get in there